src/app/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from "react";
+import styles from "./Profile.module.css";
+import NavBar from "../../components/NavBar/NavBar";
+import ProfileCard from "../../components/ProfileCard/ProfileCard";
+import { UserDog, UserSitter } from "../../../types";
+import DogIcon from "../../components/Icons/DogIcon";
+
+function Profile(): JSX.Element {
+  const [user, setUser] = useState<UserSitter | UserDog | null>(null);
+
+  useEffect(() => {
+    fetch("/api/users/me")
+      .then((response) => response.json())
+      .then((user) => setUser(user));
+  }, []);
+
+  function calculateAge(birthYear: number): number {
+    const currentYear = new Date().getFullYear();
+    return currentYear - birthYear;
+  }
+
+  if (!user) {
+    return <p>No user found</p>;
+  }
+
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <NavBar />
+      </header>
+      <main className={styles.main}>
+        <ProfileCard
```

Comment:
I would like to see a prop like `type` where the components gets the info if it is displaying a dog or sitter and sets the texts based n that